src/util/query.js

```diff
@@ -19,21 +19,18 @@ export function resolveQuery (
   query: ?string,
   extraQuery: Dictionary<string> = {}
 ): Dictionary<string> {
-  if (query) {
-    let parsedQuery
-    try {
-      parsedQuery = parseQuery(query)
-    } catch (e) {
-      process.env.NODE_ENV !== 'production' && warn(false, e.message)
-      parsedQuery = {}
-    }
-    for (const key in extraQuery) {
-      parsedQuery[key] = extraQuery[key]
-    }
-    return parsedQuery
-  } else {
-    return extraQuery
```

Comment:
Yeah, but in the other case we were skipping the whole parseQuery and for loop and returning right away the `extraQuery` instead